src/HotChocolate/Core/src/Types/Types/Descriptors/Conventions/DefaultTypeInspector.cs

```diff
@@ -711,5 +716,27 @@ private static bool IsRecord(IReadOnlyList<MemberInfo> members)
 
             return false;
         }
+
+        private bool TryGetDefaultValueFromConstructor(
```

Comment:
We need to make sure that this is really a record `IsRecord`.

Reply:
done